banditParser.py

```diff
@@ -7,7 +7,7 @@
 
 def hash(toHash):
 	hash_object = hashlib.sha256()
-	hash_object.update(toHash)
+	hash_object.update(toHash.encode('utf8'))
```

Comment:
But this might fail for Python 2, should only encode `toHash` if it's not bytes already

Reply:
Hm... I remember testing this on both py2 and py3 and there was no problem. Could you explain how would this fail for py2?

Reply:
Just being paranoid 😄 
BTW now you can completely remove bandit (approved by security) because we are moving to Snyk